File: iosApp/iosApp/features/workout/data/WorkoutScheduleEntry.swift
//
//  File.swift
//  ShapeByte
//
//  Created by Lang, Stefan [Shape Byte Tech] on 02.08.24.
//

import Foundation
import shared

// TODO: move to shared
struct WorkoutScheduleEntry: Equatable {
    let id: String
    let name: String
    let date: Date
    let progress: shared.Progress
}

File: shared/src/commonMain/kotlin/de/stefan/lang/shapebyte/features/workout/schedule/data/WorkoutScheduleEntry.kt
package de.stefan.lang.shapebyte.features.workout.schedule.data

import de.stefan.lang.coreutils.progress.Progress
import kotlinx.datetime.Instant

data class WorkoutScheduleEntry(
    val id: String,
    val name: String,
    val date: Instant,
    val progress: Progress,
)
